### PR TITLE
Added correct universal tag mappings for Russian pos_tag model

### PR DESCRIPTION
**Pull request by alvations**
Also, added unit tests for `nltk.pos_tag`.

Patching #2151

### CODE REVIEW

File: nltk/tag/__init__.py [hunk 1]
```diff
@@ -98,10 +98,18 @@ def _get_tagger(lang=None):
     return tagger
 
 
-def _pos_tag(tokens, tagset, tagger):
+def _pos_tag(tokens, tagset, tagger, lang):
```
Comment (purificant) [inline nltk/tag/__init__.py]: Is it worth keeping the signature backwards compatible and consistent with `pos_tag`, even though this is an internal method? Along the lines of having default `lang = 'eng'`

File: nltk/tag/__init__.py [hunk 2]
```diff
@@ -98,10 +98,18 @@ def _get_tagger(lang=None):
...
-        tagged_tokens = [(token, map_tag('en-ptb', tagset, tag)) for (token, tag) in tagged_tokens]
+        if lang == 'eng':
+            tagged_tokens = [(token, map_tag('en-ptb', tagset, tag)) for (token, tag) in tagged_tokens]
+        elif lang == 'rus':
+            # Note that the new Russion pos tags from the model contains suffixes,
+            # see https://github.com/nltk/nltk/issues/2151#issuecomment-430709018
+            tagged_tokens = [(token, map_tag('ru-rnc-new', tagset, tag.partition('=')[0]))
+                              for (token, tag) in tagged_tokens]
+        else:
+            raise NotImplementedError("Currently, NLTK pos_tag only supports `lang='en'`` and `lang='ru'`.")
```
Comment (purificant) [inline nltk/tag/__init__.py]: Should this be `eng` and `rus` as per `pos_tag` method signature and the code above?

File: nltk/test/unit/test_pos_tag.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for nltk.pos_tag
+"""
+
+import unittest
+
+from nltk import word_tokenize, pos_tag
+
+class TestPosTag(unittest.TestCase):
```
Comment (purificant) [inline nltk/test/unit/test_pos_tag.py:12]: Could increase code coverage by having a test for the `raise NotImplementedError` code path as well.

**Comment by purificant** [comment]
[averaged_perceptron_tagger_ru](https://github.com/nltk/nltk_data/search?q=averaged_perceptron_tagger_ru&unscoped_q=averaged_perceptron_tagger_ru) does not seem to be part of the `all` collection in [nltk_data](https://github.com/nltk/nltk_data/blob/gh-pages/collections/all.xml) which prevents the test from running. Not sure on the right approach here, should the download be added as part of travis build? or added to `all` collection?

Edit: link to tests: https://travis-ci.org/nltk/nltk/jobs/443003850

**Comment by alvations** [comment]
Thanks @purificant! Yeah, that tagger model was added without proper tests and checks. I think it should be added to `all`.

**Comment by stevenbird** [comment]
Added averaged_perceptron_tagger_ru to all in https://github.com/nltk/nltk_data/commit/2be832291f3dd7b152324cff25c56a3f9c9a812a

**Comment by alvations** [comment]
[CI: retest]

File: nltk/tag/__init__.py (hunk 1 above)
Comment (alvations) [reply nltk/tag/__init__.py]: Hmmm. I think for `pos_tag` exposed to user default should be kept but for `_pos_tag` the internal function, `lang=None` and throwing an exception if None might be easier to manage in the future.

File: nltk/tag/mapping.py [hunk 4]
```diff
@@ -75,6 +75,13 @@ def tagset_mapping(source, target):
     if source not in _MAPPINGS or target not in _MAPPINGS[source]:
         if target == 'universal':
             _load_universal_map(source)
+            # Added the new Russian National Corpus mappings because the
+            # Russian model for nltk.pos_tag() uses it.
+            _MAPPINGS['ru-rnc-new']['universal'] = {
```
Comment (BLKSerene) [inline nltk/tag/mapping.py:80]: The new mapping table which is from this pull request: https://github.com/slavpetrov/universal-pos-tags/pull/11/files is still inconsistent with the tagset listed on the official website of the Russian National Corpus.

There is no "SCONJ" POS tag in the Universal Tagset, so it must be a typo (should be "CONJ"). And there are some other errors, here is the fixed version of "ru-rnc.map" (16 tags plus "NONLEX"):

```
A	ADJ
A-PRO	PRON
ADV	ADV
ADV-PRO	PRON
ANUM	ADJ
CONJ	CONJ
INTJ	X
NONLEX	.
NUM	NUM
PARENTH	PRT
PART	PRT
PR	ADP
PRAEDIC	PRT
PRAEDIC-PRO	PRON
S	NOUN
S-PRO	PRON
V	VERB
```

And I've also generated an exhausted version of mapping table using code to match all possible combination of POS and grammatical categories listed on http://www.ruscorpora.ru/en/corpora-morph.html:

```
A	ADJ
A=m	ADJ
A=f	ADJ
A=m-f	ADJ
A=n	ADJ
A=anim	ADJ
A=inan	ADJ
A=sg	ADJ
A=pl	ADJ
A=nom	ADJ
A=gen	ADJ
A=dat	ADJ
A=acc	ADJ
A=ins	ADJ
A=ioc	ADJ
A=gen2	ADJ
A=acc2	ADJ
A=loc2	ADJ
A=voc	ADJ
A=adnum	ADJ
A=brev	ADJ
A=plen	ADJ
A=comp	ADJ
A=comp2	ADJ
A=supr	ADJ
A=pf	ADJ
A=ipf	ADJ
A=intr	ADJ
A=tran	ADJ
A=act	ADJ
A=pass	ADJ
A=med	ADJ
A=inf	ADJ
A=partcp	ADJ
A=ger	ADJ
A=indic	ADJ
A=imper	ADJ
A=imper2	ADJ
A=praet	ADJ
A=praes	ADJ
A=fut	ADJ
A=1p	ADJ
A=2p	ADJ
A=3p	ADJ
A=persn	ADJ
A=patrn	ADJ
A=famn	ADJ
A=0	ADJ
A=anom	ADJ
A=distort	ADJ
A=ciph	ADJ
A=INIT	ADJ
A=abbr	ADJ
A-PRO	PRON
A-PRO=m	PRON
A-PRO=f	PRON
A-PRO=m-f	PRON
A-PRO=n	PRON
A-PRO=anim	PRON
A-PRO=inan	PRON
A-PRO=sg	PRON
A-PRO=pl	PRON
A-PRO=nom	PRON
A-PRO=gen	PRON
A-PRO=dat	PRON
A-PRO=acc	PRON
A-PRO=ins	PRON
A-PRO=ioc	PRON
A-PRO=gen2	PRON
A-PRO=acc2	PRON
A-PRO=loc2	PRON
A-PRO=voc	PRON
A-PRO=adnum	PRON
A-PRO=brev	PRON
A-PRO=plen	PRON
A-PRO=comp	PRON
A-PRO=comp2	PRON
A-PRO=supr	PRON
A-PRO=pf	PRON
A-PRO=ipf	PRON
A-PRO=intr	PRON
A-PRO=tran	PRON
A-PRO=act	PRON
A-PRO=pass	PRON
A-PRO=med	PRON
A-PRO=inf	PRON
A-PRO=partcp	PRON
A-PRO=ger	PRON
A-PRO=indic	PRON
A-PRO=imper	PRON
A-PRO=imper2	PRON
A-PRO=praet	PRON
A-PRO=praes	PRON
A-PRO=fut	PRON
A-PRO=1p	PRON
A-PRO=2p	PRON
A-PRO=3p	PRON
A-PRO=persn	PRON
A-PRO=patrn	PRON
A-PRO=famn	PRON
A-PRO=0	PRON
A-PRO=anom	PRON
A-PRO=distort	PRON
A-PRO=ciph	PRON
A-PRO=INIT	PRON
A-PRO=abbr	PRON
ADV	ADV
ADV=m	ADV
ADV=f	ADV
ADV=m-f	ADV
ADV=n	ADV
ADV=anim	ADV
ADV=inan	ADV
ADV=sg	ADV
ADV=pl	ADV
ADV=nom	ADV
ADV=gen	ADV
ADV=dat	ADV
ADV=acc	ADV
ADV=ins	ADV
ADV=ioc	ADV
ADV=gen2	ADV
ADV=acc2	ADV
ADV=loc2	ADV
ADV=voc	ADV
ADV=adnum	ADV
ADV=brev	ADV
ADV=plen	ADV
ADV=comp	ADV
ADV=comp2	ADV
ADV=supr	ADV
ADV=pf	ADV
ADV=ipf	ADV
ADV=intr	ADV
ADV=tran	ADV
ADV=act	ADV
ADV=pass	ADV
ADV=med	ADV
ADV=inf	ADV
ADV=partcp	ADV
ADV=ger	ADV
ADV=indic	ADV
ADV=imper	ADV
ADV=imper2	ADV
ADV=praet	ADV
ADV=praes	ADV
ADV=fut	ADV
ADV=1p	ADV
ADV=2p	ADV
ADV=3p	ADV
ADV=persn	ADV
ADV=patrn	ADV
ADV=famn	ADV
ADV=0	ADV
ADV=anom	ADV
ADV=distort	ADV
ADV=ciph	ADV
ADV=INIT	ADV
ADV=abbr	ADV
ADV-PRO	PRON
ADV-PRO=m	PRON
ADV-PRO=f	PRON
ADV-PRO=m-f	PRON
ADV-PRO=n	PRON
ADV-PRO=anim	PRON
ADV-PRO=inan	PRON
ADV-PRO=sg	PRON
ADV-PRO=pl	PRON
ADV-PRO=nom	PRON
ADV-PRO=gen	PRON
ADV-PRO=dat	PRON
ADV-PRO=acc	PRON
ADV-PRO=ins	PRON
ADV-PRO=ioc	PRON
ADV-PRO=gen2	PRON
ADV-PRO=acc2	PRON
ADV-PRO=loc2	PRON
ADV-PRO=voc	PRON
ADV-PRO=adnum	PRON
ADV-PRO=brev	PRON
ADV-PRO=plen	PRON
ADV-PRO=comp	PRON
ADV-PRO=comp2	PRON
ADV-PRO=supr	PRON
ADV-PRO=pf	PRON
ADV-PRO=ipf	PRON
ADV-PRO=intr	PRON
ADV-PRO=tran	PRON
ADV-PRO=act	PRON
ADV-PRO=pass	PRON
ADV-PRO=med	PRON
ADV-PRO=inf	PRON
ADV-PRO=partcp	PRON
ADV-PRO=ger	PRON
ADV-PRO=indic	PRON
ADV-PRO=imper	PRON
ADV-PRO=imper2	PRON
ADV-PRO=praet	PRON
ADV-PRO=praes	PRON
ADV-PRO=fut	PRON
ADV-PRO=1p	PRON
ADV-PRO=2p	PRON
ADV-PRO=3p	PRON
ADV-PRO=persn	PRON
ADV-PRO=patrn	PRON
ADV-PRO=famn	PRON
ADV-PRO=0	PRON
ADV-PRO=anom	PRON
ADV-PRO=distort	PRON
ADV-PRO=ciph	PRON
ADV-PRO=INIT	PRON
ADV-PRO=abbr	PRON
ANUM	ADJ
ANUM=m	ADJ
ANUM=f	ADJ
ANUM=m-f	ADJ
ANUM=n	ADJ
ANUM=anim	ADJ
ANUM=inan	ADJ
ANUM=sg	ADJ
ANUM=pl	ADJ
ANUM=nom	ADJ
ANUM=gen	ADJ
ANUM=dat	ADJ
ANUM=acc	ADJ
ANUM=ins	ADJ
ANUM=ioc	ADJ
ANUM=gen2	ADJ
ANUM=acc2	ADJ
ANUM=loc2	ADJ
ANUM=voc	ADJ
ANUM=adnum	ADJ
ANUM=brev	ADJ
ANUM=plen	ADJ
ANUM=comp	ADJ
ANUM=comp2	ADJ
ANUM=supr	ADJ
ANUM=pf	ADJ
ANUM=ipf	ADJ
ANUM=intr	ADJ
ANUM=tran	ADJ
ANUM=act	ADJ
ANUM=pass	ADJ
ANUM=med	ADJ
ANUM=inf	ADJ
ANUM=partcp	ADJ
ANUM=ger	ADJ
ANUM=indic	ADJ
ANUM=imper	ADJ
ANUM=imper2	ADJ
ANUM=praet	ADJ
ANUM=praes	ADJ
ANUM=fut	ADJ
ANUM=1p	ADJ
ANUM=2p	ADJ
ANUM=3p	ADJ
ANUM=persn	ADJ
ANUM=patrn	ADJ
ANUM=famn	ADJ
ANUM=0	ADJ
ANUM=anom	ADJ
ANUM=distort	ADJ
ANUM=ciph	ADJ
ANUM=INIT	ADJ
ANUM=abbr	ADJ
CONJ	CONJ
CONJ=m	CONJ
CONJ=f	CONJ
CONJ=m-f	CONJ
CONJ=n	CONJ
CONJ=anim	CONJ
CONJ=inan	CONJ
CONJ=sg	CONJ
CONJ=pl	CONJ
CONJ=nom	CONJ
CONJ=gen	CONJ
CONJ=dat	CONJ
CONJ=acc	CONJ
CONJ=ins	CONJ
CONJ=ioc	CONJ
CONJ=gen2	CONJ
CONJ=acc2	CONJ
CONJ=loc2	CONJ
CONJ=voc	CONJ
CONJ=adnum	CONJ
CONJ=brev	CONJ
CONJ=plen	CONJ
CONJ=comp	CONJ
CONJ=comp2	CONJ
CONJ=supr	CONJ
CONJ=pf	CONJ
CONJ=ipf	CONJ
CONJ=intr	CONJ
CONJ=tran	CONJ
CONJ=act	CONJ
CONJ=pass	CONJ
CONJ=med	CONJ
CONJ=inf	CONJ
CONJ=partcp	CONJ
CONJ=ger	CONJ
CONJ=indic	CONJ
CONJ=imper	CONJ
CONJ=imper2	CONJ
CONJ=praet	CONJ
CONJ=praes	CONJ
CONJ=fut	CONJ
CONJ=1p	CONJ
CONJ=2p	CONJ
CONJ=3p	CONJ
CONJ=persn	CONJ
CONJ=patrn	CONJ
CONJ=famn	CONJ
CONJ=0	CONJ
CONJ=anom	CONJ
CONJ=distort	CONJ
CONJ=ciph	CONJ
CONJ=INIT	CONJ
CONJ=abbr	CONJ
INTJ	X
INTJ=m	X
INTJ=f	X
INTJ=m-f	X
INTJ=n	X
INTJ=anim	X
INTJ=inan	X
INTJ=sg	X
INTJ=pl	X
INTJ=nom	X
INTJ=gen	X
INTJ=dat	X
INTJ=acc	X
INTJ=ins	X
INTJ=ioc	X
INTJ=gen2	X
INTJ=acc2	X
INTJ=loc2	X
INTJ=voc	X
INTJ=adnum	X
INTJ=brev	X
INTJ=plen	X
INTJ=comp	X
INTJ=comp2	X
INTJ=supr	X
INTJ=pf	X
INTJ=ipf	X
INTJ=intr	X
INTJ=tran	X
INTJ=act	X
INTJ=pass	X
INTJ=med	X
INTJ=inf	X
INTJ=partcp	X
INTJ=ger	X
INTJ=indic	X
INTJ=imper	X
INTJ=imper2	X
INTJ=praet	X
INTJ=praes	X
INTJ=fut	X
INTJ=1p	X
INTJ=2p	X
INTJ=3p	X
INTJ=persn	X
INTJ=patrn	X
INTJ=famn	X
INTJ=0	X
INTJ=anom	X
INTJ=distort	X
INTJ=ciph	X
INTJ=INIT	X
INTJ=abbr	X
NONLEX	.
NONLEX=m	.
NONLEX=f	.
NONLEX=m-f	.
NONLEX=n	.
NONLEX=anim	.
NONLEX=inan	.
NONLEX=sg	.
NONLEX=pl	.
NONLEX=nom	.
NONLEX=gen	.
NONLEX=dat	.
NONLEX=acc	.
NONLEX=ins	.
NONLEX=ioc	.
NONLEX=gen2	.
NONLEX=acc2	.
NONLEX=loc2	.
NONLEX=voc	.
NONLEX=adnum	.
NONLEX=brev	.
NONLEX=plen	.
NONLEX=comp	.
NONLEX=comp2	.
NONLEX=supr	.
NONLEX=pf	.
NONLEX=ipf	.
NONLEX=intr	.
NONLEX=tran	.
NONLEX=act	.
NONLEX=pass	.
NONLEX=med	.
NONLEX=inf	.
NONLEX=partcp	.
NONLEX=ger	.
NONLEX=indic	.
NONLEX=imper	.
NONLEX=imper2	.
NONLEX=praet	.
NONLEX=praes	.
NONLEX=fut	.
NONLEX=1p	.
NONLEX=2p	.
NONLEX=3p	.
NONLEX=persn	.
NONLEX=patrn	.
NONLEX=famn	.
NONLEX=0	.
NONLEX=anom	.
NONLEX=distort	.
NONLEX=ciph	.
NONLEX=INIT	.
NONLEX=abbr	.
NUM	NUM
NUM=m	NUM
NUM=f	NUM
NUM=m-f	NUM
NUM=n	NUM
NUM=anim	NUM
NUM=inan	NUM
NUM=sg	NUM
NUM=pl	NUM
NUM=nom	NUM
NUM=gen	NUM
NUM=dat	NUM
NUM=acc	NUM
NUM=ins	NUM
NUM=ioc	NUM
NUM=gen2	NUM
NUM=acc2	NUM
NUM=loc2	NUM
NUM=voc	NUM
NUM=adnum	NUM
NUM=brev	NUM
NUM=plen	NUM
NUM=comp	NUM
NUM=comp2	NUM
NUM=supr	NUM
NUM=pf	NUM
NUM=ipf	NUM
NUM=intr	NUM
NUM=tran	NUM
NUM=act	NUM
NUM=pass	NUM
NUM=med	NUM
NUM=inf	NUM
NUM=partcp	NUM
NUM=ger	NUM
NUM=indic	NUM
NUM=imper	NUM
NUM=imper2	NUM
NUM=praet	NUM
NUM=praes	NUM
NUM=fut	NUM
NUM=1p	NUM
NUM=2p	NUM
NUM=3p	NUM
NUM=persn	NUM
NUM=patrn	NUM
NUM=famn	NUM
NUM=0	NUM
NUM=anom	NUM
NUM=distort	NUM
NUM=ciph	NUM
NUM=INIT	NUM
NUM=abbr	NUM
PARENTH	PRT
PARENTH=m	PRT
PARENTH=f	PRT
PARENTH=m-f	PRT
PARENTH=n	PRT
PARENTH=anim	PRT
PARENTH=inan	PRT
PARENTH=sg	PRT
PARENTH=pl	PRT
PARENTH=nom	PRT
PARENTH=gen	PRT
PARENTH=dat	PRT
PARENTH=acc	PRT
PARENTH=ins	PRT
PARENTH=ioc	PRT
PARENTH=gen2	PRT
PARENTH=acc2	PRT
PARENTH=loc2	PRT
PARENTH=voc	PRT
PARENTH=adnum	PRT
PARENTH=brev	PRT
PARENTH=plen	PRT
PARENTH=comp	PRT
PARENTH=comp2	PRT
PARENTH=supr	PRT
PARENTH=pf	PRT
PARENTH=ipf	PRT
PARENTH=intr	PRT
PARENTH=tran	PRT
PARENTH=act	PRT
PARENTH=pass	PRT
PARENTH=med	PRT
PARENTH=inf	PRT
PARENTH=partcp	PRT
PARENTH=ger	PRT
PARENTH=indic	PRT
PARENTH=imper	PRT
PARENTH=imper2	PRT
PARENTH=praet	PRT
PARENTH=praes	PRT
PARENTH=fut	PRT
PARENTH=1p	PRT
PARENTH=2p	PRT
PARENTH=3p	PRT
PARENTH=persn	PRT
PARENTH=patrn	PRT
PARENTH=famn	PRT
PARENTH=0	PRT
PARENTH=anom	PRT
PARENTH=distort	PRT
PARENTH=ciph	PRT
PARENTH=INIT	PRT
PARENTH=abbr	PRT
PART	PRT
PART=m	PRT
PART=f	PRT
PART=m-f	PRT
PART=n	PRT
PART=anim	PRT
PART=inan	PRT
PART=sg	PRT
PART=pl	PRT
PART=nom	PRT
PART=gen	PRT
PART=dat	PRT
PART=acc	PRT
PART=ins	PRT
PART=ioc	PRT
PART=gen2	PRT
PART=acc2	PRT
PART=loc2	PRT
PART=voc	PRT
PART=adnum	PRT
PART=brev	PRT
PART=plen	PRT
PART=comp	PRT
PART=comp2	PRT
PART=supr	PRT
PART=pf	PRT
PART=ipf	PRT
PART=intr	PRT
PART=tran	PRT
PART=act	PRT
PART=pass	PRT
PART=med	PRT
PART=inf	PRT
PART=partcp	PRT
PART=ger	PRT
PART=indic	PRT
PART=imper	PRT
PART=imper2	PRT
PART=praet	PRT
PART=praes	PRT
PART=fut	PRT
PART=1p	PRT
PART=2p	PRT
PART=3p	PRT
PART=persn	PRT
PART=patrn	PRT
PART=famn	PRT
PART=0	PRT
PART=anom	PRT
PART=distort	PRT
PART=ciph	PRT
PART=INIT	PRT
PART=abbr	PRT
PR	ADP
PR=m	ADP
PR=f	ADP
PR=m-f	ADP
PR=n	ADP
PR=anim	ADP
PR=inan	ADP
PR=sg	ADP
PR=pl	ADP
PR=nom	ADP
PR=gen	ADP
PR=dat	ADP
PR=acc	ADP
PR=ins	ADP
PR=ioc	ADP
PR=gen2	ADP
PR=acc2	ADP
PR=loc2	ADP
PR=voc	ADP
PR=adnum	ADP
PR=brev	ADP
PR=plen	ADP
PR=comp	ADP
PR=comp2	ADP
PR=supr	ADP
PR=pf	ADP
PR=ipf	ADP
PR=intr	ADP
PR=tran	ADP
PR=act	ADP
PR=pass	ADP
PR=med	ADP
PR=inf	ADP
PR=partcp	ADP
PR=ger	ADP
PR=indic	ADP
PR=imper	ADP
PR=imper2	ADP
PR=praet	ADP
PR=praes	ADP
PR=fut	ADP
PR=1p	ADP
PR=2p	ADP
PR=3p	ADP
PR=persn	ADP
PR=patrn	ADP
PR=famn	ADP
PR=0	ADP
PR=anom	ADP
PR=distort	ADP
PR=ciph	ADP
PR=INIT	ADP
PR=abbr	ADP
PRAEDIC	PRT
PRAEDIC=m	PRT
PRAEDIC=f	PRT
PRAEDIC=m-f	PRT
PRAEDIC=n	PRT
PRAEDIC=anim	PRT
PRAEDIC=inan	PRT
PRAEDIC=sg	PRT
PRAEDIC=pl	PRT
PRAEDIC=nom	PRT
PRAEDIC=gen	PRT
PRAEDIC=dat	PRT
PRAEDIC=acc	PRT
PRAEDIC=ins	PRT
PRAEDIC=ioc	PRT
PRAEDIC=gen2	PRT
PRAEDIC=acc2	PRT
PRAEDIC=loc2	PRT
PRAEDIC=voc	PRT
PRAEDIC=adnum	PRT
PRAEDIC=brev	PRT
PRAEDIC=plen	PRT
PRAEDIC=comp	PRT
PRAEDIC=comp2	PRT
PRAEDIC=supr	PRT
PRAEDIC=pf	PRT
PRAEDIC=ipf	PRT
PRAEDIC=intr	PRT
PRAEDIC=tran	PRT
PRAEDIC=act	PRT
PRAEDIC=pass	PRT
PRAEDIC=med	PRT
PRAEDIC=inf	PRT
PRAEDIC=partcp	PRT
PRAEDIC=ger	PRT
PRAEDIC=indic	PRT
PRAEDIC=imper	PRT
PRAEDIC=imper2	PRT
PRAEDIC=praet	PRT
PRAEDIC=praes	PRT
PRAEDIC=fut	PRT
PRAEDIC=1p	PRT
PRAEDIC=2p	PRT
PRAEDIC=3p	PRT
PRAEDIC=persn	PRT
PRAEDIC=patrn	PRT
PRAEDIC=famn	PRT
PRAEDIC=0	PRT
PRAEDIC=anom	PRT
PRAEDIC=distort	PRT
PRAEDIC=ciph	PRT
PRAEDIC=INIT	PRT
PRAEDIC=abbr	PRT
PRAEDIC-PRO	PRON
PRAEDIC-PRO=m	PRON
PRAEDIC-PRO=f	PRON
PRAEDIC-PRO=m-f	PRON
PRAEDIC-PRO=n	PRON
PRAEDIC-PRO=anim	PRON
PRAEDIC-PRO=inan	PRON
PRAEDIC-PRO=sg	PRON
PRAEDIC-PRO=pl	PRON
PRAEDIC-PRO=nom	PRON
PRAEDIC-PRO=gen	PRON
PRAEDIC-PRO=dat	PRON
PRAEDIC-PRO=acc	PRON
PRAEDIC-PRO=ins	PRON
PRAEDIC-PRO=ioc	PRON
PRAEDIC-PRO=gen2	PRON
PRAEDIC-PRO=acc2	PRON
PRAEDIC-PRO=loc2	PRON
PRAEDIC-PRO=voc	PRON
PRAEDIC-PRO=adnum	PRON
PRAEDIC-PRO=brev	PRON
PRAEDIC-PRO=plen	PRON
PRAEDIC-PRO=comp	PRON
PRAEDIC-PRO=comp2	PRON
PRAEDIC-PRO=supr	PRON
PRAEDIC-PRO=pf	PRON
PRAEDIC-PRO=ipf	PRON
PRAEDIC-PRO=intr	PRON
PRAEDIC-PRO=tran	PRON
PRAEDIC-PRO=act	PRON
PRAEDIC-PRO=pass	PRON
PRAEDIC-PRO=med	PRON
PRAEDIC-PRO=inf	PRON
PRAEDIC-PRO=partcp	PRON
PRAEDIC-PRO=ger	PRON
PRAEDIC-PRO=indic	PRON
PRAEDIC-PRO=imper	PRON
PRAEDIC-PRO=imper2	PRON
PRAEDIC-PRO=praet	PRON
PRAEDIC-PRO=praes	PRON
PRAEDIC-PRO=fut	PRON
PRAEDIC-PRO=1p	PRON
PRAEDIC-PRO=2p	PRON
PRAEDIC-PRO=3p	PRON
PRAEDIC-PRO=persn	PRON
PRAEDIC-PRO=patrn	PRON
PRAEDIC-PRO=famn	PRON
PRAEDIC-PRO=0	PRON
PRAEDIC-PRO=anom	PRON
PRAEDIC-PRO=distort	PRON
PRAEDIC-PRO=ciph	PRON
PRAEDIC-PRO=INIT	PRON
PRAEDIC-PRO=abbr	PRON
S	NOUN
S=m	NOUN
S=f	NOUN
S=m-f	NOUN
S=n	NOUN
S=anim	NOUN
S=inan	NOUN
S=sg	NOUN
S=pl	NOUN
S=nom	NOUN
S=gen	NOUN
S=dat	NOUN
S=acc	NOUN
S=ins	NOUN
S=ioc	NOUN
S=gen2	NOUN
S=acc2	NOUN
S=loc2	NOUN
S=voc	NOUN
S=adnum	NOUN
S=brev	NOUN
S=plen	NOUN
S=comp	NOUN
S=comp2	NOUN
S=supr	NOUN
S=pf	NOUN
S=ipf	NOUN
S=intr	NOUN
S=tran	NOUN
S=act	NOUN
S=pass	NOUN
S=med	NOUN
S=inf	NOUN
S=partcp	NOUN
S=ger	NOUN
S=indic	NOUN
S=imper	NOUN
S=imper2	NOUN
S=praet	NOUN
S=praes	NOUN
S=fut	NOUN
S=1p	NOUN
S=2p	NOUN
S=3p	NOUN
S=persn	NOUN
S=patrn	NOUN
S=famn	NOUN
S=0	NOUN
S=anom	NOUN
S=distort	NOUN
S=ciph	NOUN
S=INIT	NOUN
S=abbr	NOUN
S-PRO	PRON
S-PRO=m	PRON
S-PRO=f	PRON
S-PRO=m-f	PRON
S-PRO=n	PRON
S-PRO=anim	PRON
S-PRO=inan	PRON
S-PRO=sg	PRON
S-PRO=pl	PRON
S-PRO=nom	PRON
S-PRO=gen	PRON
S-PRO=dat	PRON
S-PRO=acc	PRON
S-PRO=ins	PRON
S-PRO=ioc	PRON
S-PRO=gen2	PRON
S-PRO=acc2	PRON
S-PRO=loc2	PRON
S-PRO=voc	PRON
S-PRO=adnum	PRON
S-PRO=brev	PRON
S-PRO=plen	PRON
S-PRO=comp	PRON
S-PRO=comp2	PRON
S-PRO=supr	PRON
S-PRO=pf	PRON
S-PRO=ipf	PRON
S-PRO=intr	PRON
S-PRO=tran	PRON
S-PRO=act	PRON
S-PRO=pass	PRON
S-PRO=med	PRON
S-PRO=inf	PRON
S-PRO=partcp	PRON
S-PRO=ger	PRON
S-PRO=indic	PRON
S-PRO=imper	PRON
S-PRO=imper2	PRON
S-PRO=praet	PRON
S-PRO=praes	PRON
S-PRO=fut	PRON
S-PRO=1p	PRON
S-PRO=2p	PRON
S-PRO=3p	PRON
S-PRO=persn	PRON
S-PRO=patrn	PRON
S-PRO=famn	PRON
S-PRO=0	PRON
S-PRO=anom	PRON
S-PRO=distort	PRON
S-PRO=ciph	PRON
S-PRO=INIT	PRON
S-PRO=abbr	PRON
V	VERB
V=m	VERB
V=f	VERB
V=m-f	VERB
V=n	VERB
V=anim	VERB
V=inan	VERB
V=sg	VERB
V=pl	VERB
V=nom	VERB
V=gen	VERB
V=dat	VERB
V=acc	VERB
V=ins	VERB
V=ioc	VERB
V=gen2	VERB
V=acc2	VERB
V=loc2	VERB
V=voc	VERB
V=adnum	VERB
V=brev	VERB
V=plen	VERB
V=comp	VERB
V=comp2	VERB
V=supr	VERB
V=pf	VERB
V=ipf	VERB
V=intr	VERB
V=tran	VERB
V=act	VERB
V=pass	VERB
V=med	VERB
V=inf	VERB
V=partcp	VERB
V=ger	VERB
V=indic	VERB
V=imper	VERB
V=imper2	VERB
V=praet	VERB
V=praes	VERB
V=fut	VERB
V=1p	VERB
V=2p	VERB
V=3p	VERB
V=persn	VERB
V=patrn	VERB
V=famn	VERB
V=0	VERB
V=anom	VERB
V=distort	VERB
V=ciph	VERB
V=INIT	VERB
V=abbr	VERB
```

Comment (alvations) [reply nltk/tag/mapping.py:80]: @BLKSerene Thanks I've updated the mappings accordingly.

**Comment by alvations** [comment]
[CI: retest]

**Comment by alvations** [comment]
[CI: retest]

**Comment by stevenbird** [comment]
Thanks @alvations, @BLKSerene!

**Pull request merged**